pkg/clusterbuilder/providers/capvcd/values/cluster_values.yaml

```diff
@@ -8,6 +8,9 @@ global:
         - "10.205.105.253"
     proxy:
       enabled: true
+      httpProxy: "http://10.205.105.253:3128"
+      httpsProxy: "http://10.205.105.253:3128"
+      noProxy: "test.gigantic.io,10.205.105.0/24,cd.neoedge.cloud"
```

Comment:
From my experience most applications do not support CIDRs in the `no_proxy` variable. This only works for domains and plain IP addresses.

Reply:
> From my experience most applications do not support CIDRs in the `no_proxy` variable. This only works for domains and plain IP addresses.

we've been using CIDRs in noproxy vars for at least 2 years with Panamax with no problems.

Reply:
that being said, it's rare that we would need to connect to node IPs anyway so this is mostly just insurance (that subnet is the node subnet for `gerbil`)

Reply:
Wild. I've rarely seen any binaries using it like this in the past. That's also the reason for several StackExchange threads. 😅

Reply:
curl has supported it for a couple of years now (according to the release notes anyway) so any binary using libcurl will automatically inherit this ability too. it's definitely a bit of an edge case though.

Reply:
If this has been tested and is actually working: All good. It's just my experience and several similar results when googling "no_proxy CIDR" that made me stumble.

If it is not proven to work, I would not add it as people might expect it to work and therefore won't double check in case of an incident.